### PR TITLE
20.5.0 md

### PR DESCRIPTION
**Pull request by jochman**
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.5.0.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+- __PcapHTTPExtractor__
+Fixed an issue where a pcap file which contains only an HTTP response was mishandled.
+- __SandboxDetonateFile__
+Fixed an issue when some fields are missing from the response.
+- __QRadarGetCorrelationLogs__
+- Use the **QRadarCorrelationLog** playbook instead.
+
+
+#### 2 Deprecated Scripts
+- __Elasticsearch__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:199]: why is this here?

File: content-repo/extra-docs/releases/20.5.0.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+  - Fixed an issue where the script attempted to create indicators with *Attachment* type instead of *File* type.
+- __PcapHTTPExtractor__
+Fixed an issue where a pcap file which contains only an HTTP response was mishandled.
+- __SandboxDetonateFile__
+Fixed an issue when some fields are missing from the response.
+- __QRadarGetCorrelationLogs__
+- Use the **QRadarCorrelationLog** playbook instead.
+
+
+#### 2 Deprecated Scripts
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:198]: there are more than 2 here

File: content-repo/extra-docs/releases/20.5.0.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+Removed default list names and updated playbook logic.
+- __TIM - Process Indicators - Fully Automated__
+Fixed task name and score.
+- __Phishing Investigation - Generic v2__
+The playbook now uses Block Indicators - Generic v2 to block malicious indicators (off by default).
+- __Impossible Traveler__
+  - Simplified the process that gets details of the user's manager.
+  - Fixed a potential error with running Active Directory commands when the integration is disabled.
+
+#### Deprecated Playbook
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:298]:
```suggestion
#### 3 Deprecated Playbooks
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+Use the 'Block Indicators - Generic v2' playbook instead.
+- __Hunt for bad IOCs__
+Use the Search Endpoints By Hash playbook instead.
+- __Rapid IOC Hunting Playbook__
+Use the Hunt File Hash playbook instead.
+
+---
+### Dashboards
+
+####  New Dashboards
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:309]:
```suggestion
####  New Dashboard
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+- __Employee Health Status__
+
+
+---
+### Incident Fields
+- __What are the areas that need improvement?__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:315]: are these new? need to add a `#### New Incident Fields` title

File: content-repo/extra-docs/releases/20.5.0.md [hunk 6]
```diff
@@ -0,0 +1,373 @@
...
+Added edit layout for the **Indicator Feed** incident type.
+- __GDPR Data Breach - Summary__
+Changed "Date breach information" section from rows to cards.
+- __GCP Compute Engine Misconfiguration - Summary__
+New layout for 'GCP Compute Engine Misconfiguration'.
+
+
+---
+### Reputations
+  - The **CVE CVSS Score** indicator type is only available in Demisto version 4.5.x and earlier.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:373]: as @gshriki asked, lets check what's this about, it looks weird

File: content-repo/extra-docs/releases/20.5.0.md [hunk 7]
```diff
@@ -0,0 +1,373 @@
+## Demisto Content Release Notes for version 20.5.0 (52248)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.0.md:1]:
```suggestion
## Cortex XSOAR Content Release Notes for version 20.5.0 (52248)
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 8]
```diff
@@ -0,0 +1,373 @@
...
+- __IBM QRadar__
+- Fixed an issue where the test module did not work as expected.
+- __Microsoft Graph User__
+Added the ***msgraph-direct-reports*** command, which retrieves a user's direct reports.
+- __Lockpath KeyLight v2__
+- Added *kl-get-user-by-id* command.
+- __Mail Sender (New)__
+Added support for the *SSL/TLS* parameter to configure SSL/TLS connection, which is not STARTTLS.
+- __GitHub__
+- Added handling for deleted forked repositories in the ***GitHub-get-pull-request*** command.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.5.0.md:143]:
```suggestion
Added handling for deleted forked repositories in the ***GitHub-get-pull-request*** command.
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 9]
```diff
@@ -0,0 +1,373 @@
...
+- __Palo Alto Networks PAN-OS EDL Service__
+  - Removed *Long Running Instance* from instance configuration.
+  - Set the listener host to 0.0.0.0 in order to handle IPv6.
+- __TAXII Server__
+  - Removed *Long Running Instance* from instance configuration.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.5.0.md:148]:
```suggestion
 Removed *Long Running Instance* from instance configuration.
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 10]
```diff
@@ -0,0 +1,373 @@
...
+- __PenteraOperationToIncident__
+Groups Pentera Full Actions Reports by Operation Type to generate an output usable in creating incidents.
+- __SetGridField__
+Creates a Grid table from items or key-value pairs.
+
+####  6 Improved Scripts
+- __DBotPredictPhishingWords__
+Added the option to map automation output to out-of-the-box incidents fields.
+- __SanePdfReports__
+- Fixed logos usage and added failure verbose output.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.5.0.md:186]:
```suggestion
Fixed logos usage and added failure verbose output.
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 11]
```diff
@@ -0,0 +1,373 @@
...
+- __PhishLabsPopulateIndicators__
+  - Fixed an issue where email indicators were not classified correctly.
+  - Fixed an issue where the script attempted to create indicators with *Attachment* type instead of *File* type.
+- __PcapHTTPExtractor__
+Fixed an issue where a pcap file which contains only an HTTP response was mishandled.
+- __SandboxDetonateFile__
+Fixed an issue when some fields are missing from the response.
+- __QRadarGetCorrelationLogs__
+- Use the **QRadarCorrelationLog** playbook instead.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.5.0.md:195]:
```suggestion
Use the **QRadarCorrelationLog** playbook instead.
```

File: content-repo/extra-docs/releases/20.5.0.md [hunk 12]
```diff
@@ -0,0 +1,373 @@
...
+- __Endace__
+  - The EndaceProbe Analytics Platform provides 100% accurate, continuous packet capture on network links up to 100Gbps, with unparalleled depth of storage and retrieval performance.  Coupled with the Endace InvestigationManager, this provides a central search and data-mining capability across a fabric of EndaceProbes deployed in a network.
+  - This integration uses Endace APIs to search, archive and download PCAP file from either a single EndaceProbe or many via the InvestigationManager and enables integration of full historical packet capture into security automation workflows.
+- __Maltiverse__
+Use the Maltiverse integration to analyze suspicious hashes, URLs, domains and IP addresses.
+- __Malwarebytes__
+Scan and Remediate threats on endpoints in the Malwarebytes cloud.
+- __MITRE ATT&CK__
+MITRE ATT&CK® is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations. The ATT&CK knowledge base is used as a foundation for the development of specific threat models and methodologies in the private sector, in government, and in the cybersecurity product and service community.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.5.0.md:26]: @kirbles19 Please review:
```suggestion
Fetch MITRE’s Adversarial Tactics, Techniques, and Common Knowledge (ATT&CK®) content via a feed. MITRE ATT&CK® is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations. The ATT&CK knowledge base is used as a foundation for the development of specific threat models and methodologies in the private sector, in government, and in the cybersecurity product and service community.
```

File: content-repo/extra-docs/releases/20.5.0.md (hunk 6 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.5.0.md:373]: it's from 3 months ago. Probably upon the refactor. deleting

**Comment by glicht** [comment]
Preview of the release notes is available at: https://deploy-preview-114--demisto-content-docs.netlify.app/docs/reference/releases/20.5.0
